cfme/base/ui.py

```diff
@@ -1081,7 +1080,7 @@ def step(self):
 
 
 @navigator.register(Region)
-class Advanced(CFMENavigateStep):
+class AdvancedRegion(CFMENavigateStep):
```

Comment:
I think we need to register it with `Advanced` short name against `Region`
```
@navigator.register(Region, "Advanced")
```

Reply:
oh, good point :+1: